src/inc/msquic.h

```diff
@@ -909,6 +909,9 @@ typedef struct QUIC_SCHANNEL_CREDENTIAL_ATTRIBUTE_W {
 #define QUIC_PARAM_CONN_STATISTICS_V2                   0x05000016  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_STATISTICS_V2_PLAT              0x05000017  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_ORIG_DEST_CID                   0x05000018  // uint8_t[]
+#define QUIC_PARAM_CONN_ADD_LOCAL_ADDRESS               0x05000019  // QUIC_ADDR
+#define QUIC_PARAM_CONN_REMOVE_LOCAL_ADDRESS            0x0500001A  // QUIC_ADDR
+#define QUIC_PARAM_CONN_LOCAL_UNUSED_DEST_CID_COUNT     0x0500001B  // uint16_t
```

Comment:
Is this for test only? Do real clients really need this? If it's test only, please put in `msquicp.h`.

Reply:
Real clients also need this.

Reply:
I'd really prefer to be able to abstract away the complexities of connection IDs from clients. Why exactly do clients need this information? To know if they can add a new path?

Reply:
Yes. This information is needed to know when clients can add a new local address. But this is no longer needed if we adopt a new way in which clients defer sending a path challenge until they receive a new ConID.